env-defaults

```diff
@@ -158,6 +168,9 @@ REAPABLE_WORK_AGE_MINUTES=1440
 # The time (in seconds) between invocations of the work failer service
 WORK_FAILER_PERIOD_SEC=120
 
+# The time (in seconds) between invocations of the work-item update queue processor
+WORK_ITEM_UPDATE_QUEUE_PROCESSOR_PERIOD_SEC=1
```

Comment:
Ok I was looking at the logs it looks like things take 21 seconds when there is nothing to pull. So we have long polling of 20 seconds and then this 1 second delay where we won't pick up any messages. I don't really see any reason for ever delaying things so I think I'd like to just see this config removed and not have the threads processing the queues ever call sleep since we're using long polling.

Reply:
Fixed